docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,45 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v2/users?offset=0&limit=10&guests=false
+
+including an ``access_token`` of a server admin.
+The parameters ``offset`` and ``limit`` are required only for pagination.
+Per default a ``limit`` of 100 is used. If the endpoint returns less entries
+than specified by ``limit`` then there are no more users left.
+The parameter ``name`` can be used to filter by user name.
+The parameter ``guests`` can be used to exclude guest users.
+The parameter ``deactivated`` can be used to include deactivated users.
+
+It returns a JSON body like the following:
+
+.. code:: json
```

Comment:
this isn't coming out in the formatted file (https://github.com/Awesome-Technologies/synapse/blob/user_admin/docs/admin_api/user_admin_api.rst): possibly it needs an extra newline or something like that?